spec/features/managing_step_by_step_pages_spec.rb

```diff
@@ -680,6 +697,17 @@ def and_the_prompt_should_link_to_the_check_links_page
     end
   end
 
+  def but_when_I_edit_a_step
```

Comment:
As mentioned abveI think this can be removed. 
There are already steps in `managing_steps_spec.rb` that test editing a step.